game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -53,6 +53,19 @@ object TeamDAO extends TableQuery(new Teams(_)) {
     )
   }
 
+  def updateScore(db: Database, team_id: Int, value: Int): Unit ={
```

Comment:
Could we rename to `addScore` or similar? `updateScore` makes it feel like it is setting its value.